.github/workflows/intel.yml

```
@@ -157,8 +157,8 @@ jobs:
         set +e
         source /opt/intel/oneapi/setvars.sh
         set -e
-        export CXX=$(which dpcpp)
-        export CC=$(which clang)
+        export CXX=$(which icpx)
+        export CC=$(which icx)
```

But should actually be added in CMake, for now here:
```suggestion
        export CC=$(which icx)
        export CXXFLAGS="-fsycl ${CXXFLAGS}"
```

---

Since we use `-DWarpX_COMPUTE=SYCL`, `-fsycl` will be added in the cmake setup in amrex.  But I am not sure if this is explicitly needed when compiling OpenPMD.